kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Coordinator.java

```diff
@@ -255,6 +288,77 @@ private void commitToTable(
     }
   }
 
+  private static String dataFilePath(DataFile dataFile) {
+    return dataFile.path().toString();
+  }
+
+  private static String deleteFilePath(DeleteFile deleteFile) {
+    return deleteFile.path().toString();
+  }
+
+  private <T> List<T> deduplicateBatch(
+      List<T> files,
+      Function<T, String> getPathFn,
+      String fileType,
+      UUID commitId,
+      TableIdentifier tableIdentifier) {
+    return deduplicate(
+        files,
+        getPathFn,
+        (numDuplicatedFiles, path) ->
+            String.format(
+                "Detected %d %s files with the same path=%s across payloads during commitId=%s for table=%s",
```

Comment:
If we see this log message, it means either:
- The coordinator has somehow ingested the same kafka message twice
- The workers have somehow sent a CommitResponse twice

If I have time, I'll see if I can further disambiguate between those two cases (involves changes a little higher up).

Reply:
Should we log more of the event details? Such as the event.id(), event.groupId(), event.timestamp()?

Reply:
Added it all to this log line. 
I have not added it to the other log lines because it's complex to keep track of that information and I'd rather not make this code any more complicated than it already is. 
I think it should be fine for the other log lines; topic partition and offset go a long way (we can just pull the message from kafka, deserialize it and get the same information).